src/System.IO.Pipes/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,4 +1,5 @@
 # Exposed public in System.IO.Pipes.AccessControl but implemented in System.IO.Pipes
+TypesMustExist : Type 'System.IO.Pipes.AnonymousPipeServerStreamAcl' does not exist in the reference but it does exist in the implementation.
```

Comment:
This should be Named. I am not really sure if this is required, should I keep it?

Reply:
The name of your class and your ref assembly don't match.

Reply:
Scratch that, sorry, I assume you didn't remove your other implementation when you built? You shouldn't have it in here.